README.md

```diff
@@ -90,7 +90,7 @@ In some cases, you may want to ignore exception-related errors as per class basi
 parameters:
 	exceptionRules:
 		methodWhitelist:
-			PHPUnit\Framework\TestCase: '#^(setup|setupbeforeclass|teardown|teardownafterclass|test.+)$#i'
+			PHPUnit\Framework\TestCase: '#^(setup|setupbeforeclass|teardown|teardownafterclass|test.*)$#i'
```

Comment:
should the method `public function test()` be considered a test?

Reply:
Yes, I've tested it

Reply:
I'm not sure about it, but if it does, the `.*` is not needed as we're already matching methods starting with at least one of these prefixes.

Reply:
(the `$` should also be left out)

Reply:
ok, so the more correct would be `#^(test|setup|setupbeforeclass|teardown|teardownafterclass)#i`

Reply:
#^test|(setup|setupbeforeclass|teardown|teardownafterclass)$#i

Reply:
The more correct would be `#^test|^(setup|setupbeforeclass|teardown|teardownafterclass)$#i

Reply:
@pepakriz how could we provide a default value so that you don't need to repeat it on every project?

Reply:
Your example is more optimized but less readable for me. I don't care.

Reply:
I'm not a fan of universal presets. Every project is unique and needs different setup and rules. But I'm open to suggestions

Reply:
the performance penalty over the `.*` cannot be ignored. How about `#^(test|(setup|setupbeforeclass|teardown|teardownafterclass)$)#i ?

Reply:
Much better, I'll change it